modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -24,6 +24,57 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
     if (!empty($this->configuration['gallery_display'])) {
       $behavior['gallery_display'] = $this->configuration['gallery_display'];
     }
+    if (!empty($this->configuration['card_width'])) {
+      $behavior['card_width'] = $this->configuration['card_width'];
+    }
+    if (!empty($this->configuration['card_style'])) {
+      $behavior['card_style'] = $this->configuration['card_style'];
+    }
+    if (!empty($this->configuration['card_width_sm'])) {
+      $behavior['az_display_settings']['card_width_sm'] = $this->configuration['card_width_sm'];
+    }
+    if (!empty($this->configuration['card_width_xs'])) {
+      $behavior['az_display_settings']['card_width_xs'] = $this->configuration['card_width_xs'];
+    }
+    // Getting the card width checking field.
+    if (!empty($this->configuration['card_width_field'])) {
+      $card_width_field = $row->getSourceProperty($this->configuration['card_width_field']);
+      $card_count_field = $row->getSourceProperty($this->configuration['card_count']);
```

Comment:
I think it would be good to name the `card_count` configuration property `card_count_field` for consistency.
```suggestion
      $card_count_field = $row->getSourceProperty($this->configuration['card_count_field']);
```